Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'link_header'
 gem 'mail'
 gem 'mailjet'
 gem 'mas-assets', git: 'git@github.com:moneyadviceservice/mas-assets'
-gem 'meta-tags', github: 'moneyadviceservice/meta-tags'
+gem 'meta-tags', github: 'kpumuk/meta-tags', branch: 'master'
```

Comment:
Good to see us get rid of our fork, but can we not just switch to the published gem then?